scss/fonts.scss

```diff
@@ -1,3 +1,7 @@
+* {
```

Comment:
Remember this should be in `styles.scss` file

Reply:
Don't know how that get there T_T